script/post-upgrade

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 
+deb-systemd-invoke restart procps.service
```

Comment:
Could also use `sysctl` directly.

Reply:
Just following best practices. `/etc/sysctl.d/README.sysctl`:

> After making any changes, please run "service procps reload" (or, from a Debian package maintainer script "deb-systemd-invoke restart procps.service").

But i can check if `deb-systemd-invoke` is available and fallback to `sysctl`

Reply:
Sounds good. I was thinking about people e.g. on devuan or some other systemd-less system. Not sure whether they have a drop-in for `deb-systemd-invoke` to do the non-systemd equivalent (I myself am blissfully using systemd without really understanding all the fuss - probably got lucky in having been exposed to linux too late to form strong opinions there :) ).

Reply:
Done. But it seems that even the systemd hating zealots from Devuan provide deb-systemd-invoke 😛

Reply:
The check follows Debians best packing practices guide: https://www.debian.org/doc/manuals/developers-reference/best-pkging-practices.html#best-practices-for-maintainer-scripts